src/main/groovy/life/qbic/service/ILocationService.groovy

```diff
@@ -30,6 +33,18 @@ interface ILocationService {
    * @param email email address
    * @return list of Location objects
    * @since 1.0.0
+   * @deprecated As of version 1.1.0 this method should no longer be used and will be removed in the future. {@link #getLocationsForPerson} should be used instead.
    */
+  //@Deprecated(since="1.1.0", forRemoval=true) //TODO this can be enabled in Java 11
+  @Deprecated
   List<Location> getLocationsForEmail(String email)
+
+  /**
+   * List all locations attached to a person with the provided identifier.<br/>
+   * Currently one email address is used as identifier for a person in the system
+   * @param identifier the identifier for a person
+   * @return a list of locations found, if the query was successful, failure status infos otherwise
```

Comment:
```suggestion
   * @return a list of locations found
```
I think this description is obsolete now that we don't return an httpResponse object